config/locales/doorkeeper.es.yml

```diff
@@ -0,0 +1,112 @@
+---
+es:
+  activerecord:
+    attributes:
+      doorkeeper/application:
+        name: Nombre
+        redirect_uri: Redirección URI
+    errors:
+      models:
+        doorkeeper/application:
+          attributes:
+            redirect_uri:
+              fragment_present: no puede contener un fragmento.
+              invalid_uri: debe ser una URI válida.
+              relative_uri: debe ser uns URI absoluta.
+              secured_uri: debe ser una URI HTTPS/SSL.
```

Comment:
una -> un

URI Should be masculine because it's Uniform Resource Identifier. Identifier in Spanish is "Identificador", a masculine word.